src/overlays/actors/ovl_Koinobori/ac_koinobori.c

```diff
@@ -109,16 +109,22 @@ void aKOI_actor_init(Actor* thisx, Game_Play* game_play) {
 }
 
 #define OPEN_POLY_OPA_DISPS()                 \
+    do {                                      \
+    } while (0);                              \
```

Comment:
I think what darkeye was saying was only put 1 do while at the end of each macro. The OPEN_DISPS macro has an extra one at the start for matching reasons, but its unnecessary for these ones.